core/invoker/src/main/scala/whisk/core/container/package.scala

```diff
@@ -53,6 +53,14 @@ package object container {
         def duration = Duration.create(end.toEpochMilli() - start.toEpochMilli(), MILLISECONDS)
     }
 
+    object Interval {
+        /** An interval starting now with zero duration. */
+        def zero = {
```

Comment:
It matters, yes. Imagine a socket exception on `/init`. The activation should have the correct start time, even if it is a zero interval.